src/windows/TransactionDetails.tsx

```diff
@@ -24,6 +24,8 @@ import { Alert } from "../utils/alert";
 import { useTranslation } from "react-i18next";
 import { namespaces } from "../i18n/i18n.constants";
 
+const durationAsSeconds = (n: number) => `${n /1000}s`
```

Comment:
nitpick: I think it looks better if the number of decimal are max 2.
Right now `1234ms` shows up as `1.234s`.


```suggestion
const durationAsSeconds = (n: number) => `${(n/1000).toFixed(2)}s`
```